omnibus/config/projects/agent.rb

```diff
@@ -197,7 +197,7 @@
   ]
   if with_python_runtime? '2'
     # noinspection RubyLiteralArrayInspection
-    additional_sign_files_list = [
+    additional_sign_files_list <<= [
```

Comment:
Using `<<=` will append what's on the right to the array (ie. you'll have an array within the array. I believe what you want here is:
```suggestion
    additional_sign_files_list.concat [
```